### DISCUSSION TITLE
understanding metadata of data_table

### DISCUSSION THREAD

**Discussion by tradanghi1999** [dcaribou/transfermarkt-datasets#295] [Q&A]
Hi,
I'm a student from Viet Nam. I have used your dataset in my data homework in the data course.

But there are something about your dataset that I don't understand

In your club_games table there are columns (own_goal, own_position, opponent_goals, opponent_position) that I cannot understand.

For example, in line id 30756, we have data about Ancelotti and Xavi, which I can't find it meaning.

Please explain it for me. Thanks

**Comment by dcaribou** [comment]
Hi @tradanghi1999,
What do you mean by line ID? The `club_games` asset does not have an explict primary key, as far as I can tell.
Maybe you can paste the values you see that you are not understanding? You can also provide a screenshot.
